### DISCUSSION TITLE
smart_format not working for nova-2 with german as language

### DISCUSSION THREAD

**Discussion by iKushiu** [deepgram/community#541] [General help] [answered]
### Which Deepgram product are you using?

Deepgram API

### Details

When I try to transcribe German-language audio with Nova-2 and have smart_format turned on, smart_format is not used (or it doesn't work properly yet), because the output then says "vierhundertneunundsechzig" instead of 469, for example.

### If you are making a request to the Deepgram API, what is the full Deepgram URL you are making a request to?

/v1/listen?language=de&model=nova-2&smart_format=true

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

b39ca806-4c70-4ac8-8b8d-07655515e4d1

### If possible, please attach your code or paste it into the text box.

I am using the Deepgram API Playground for testing.

### If possible, please attach an example audio file to reproduce the issue.

[audio.zip](https://github.com/deepgram/community/files/13986408/audio.zip)

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @iKushiu, I'm sorry Deepgram isn't providing the formatting results you're looking for. Please refer to our [Smart Formatting](https://developers.deepgram.com/docs/smart-format) documentation, which notes the following details:

> Smart Format has the broadest support for English-language models.

> On non-English models, Smart Format will apply all available formatters for that language. For the majority of languages, this is currently limited to [Punctuation](https://developers.deepgram.com/docs/punctuation/) and [Paragraphs](https://developers.deepgram.com/docs/paragraphs/).

Our current support for German Smart Formatting is that we will add punctuation and paragraphs, but do not format numerals.

**Reply by iKushiu** [reply]
Okay, yes thanks, I must have missed that
